config/boot.rb

```diff
@@ -1,4 +1,8 @@
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
 
-require 'bundler/setup' # Set up gems listed in the Gemfile.
-require 'bootsnap/setup' # Speed up boot time by caching expensive operations.
+# if !defined?(Spring) && [nil, "development", "test"].include?(ENV["RAILS_ENV"])
+#   load File.expand_path("../bin/spring", __dir__)
+# end
```

Comment:
Can this be deleted?

https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/config/boot.rb.tt